### PR TITLE
#353 Rename hasDivider field in Strapi

### PR DESCRIPTION
**Pull request by katarinazaprazna**
- Rename `hasDivider` field into `hasDividers` so that the field becomes more self-explanatory

### CODE REVIEW

File: next/src/components/common/NavBar/NavMenu/NavMenuContent.tsx [hunk 1]
```diff
@@ -58,7 +58,11 @@ const NavMenuContent = ({ sections, seeAllLinkProps, className }: NavMenuContent
             }
 
             return (
-              <NavMenuContentCell key={cell.id} colSpan={cell.colSpan} hasDivider={cell.hasDivider}>
+              <NavMenuContentCell
+                key={cell.id}
+                colSpan={cell.colSpan}
+                hasDivider={cell.hasDividers}
```
Comment (radoslavzeman) [inline next/src/components/common/NavBar/NavMenu/NavMenuContent.tsx:64]: Shouldn't we rename also the prop name in `NavMenuContentCell` component?

**Review by radoslavzeman — APPROVED** [review]
💯

**Comment by radoslavzeman** [comment]
Is `internalName` still used?
image

File: next/src/components/common/NavBar/NavMenu/NavMenuContent.tsx (hunk 1 above)
Comment (katarinazaprazna) [reply next/src/components/common/NavBar/NavMenu/NavMenuContent.tsx:64]: Thanks for mentioning this :) I was accounting for that. However, this prop will be removed by a subsequent PR - the one which is dealing with the uniformity of dividers. So, I deemed renaming as unnecessary

**Comment by katarinazaprazna** [comment]
> Is `internalName` still used? image

@radoslavzeman It is, for internal representation of the `Date` object :) We use the same pattern for opening hours

**Pull request merged**